server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java

```diff
@@ -53,29 +54,29 @@ public static String removeTrailingSlash(String path) {
   }
 
   public static Path removeTrailingSlash(Path path) {
-    String pathStr = path.toString();
+    String pathStr = Objects.requireNonNull(path).toString();
```

Comment:
There are volume replacement ITs may be worthwhile having those exercise replacing a volume where there are tablets that are using that volume with non infinite ranges.  This could be another follow in testing issue.